app/conversations/__init__.py

```diff
@@ -32,3 +32,10 @@ def post(self, user=None):
         
         result = SocketService.create_conversation(conversation_id, user['email'], participants)
         return utils.make_response(data = { 'conversationId': conversation_id, 'participants': result })
+
+    def remove(self, user=None):
```

Comment:
Additionally, you should add a `PUT` key to the `arguments` dictionary containing the following allowed arguments: `add`, `remove`. Then, you should marshal the request as usual and add all users in the `add` list that gets sent up while removing all users in the `remove` list. 

Keep in mind that neither of these arguments are required!